src/Generated/ChatClient.cs

```diff
@@ -20,6 +23,58 @@ protected ChatClient()
 
         public ClientPipeline Pipeline { get; }
 
+        [Experimental("OPENAI001")]
+        public virtual CollectionResult GetChatCompletions(string after, int? limit, string order, IDictionary<string, string> metadata, string model, RequestOptions options)
```

Comment:
Could you re-run the Export-Api.ps1 script to re-generate the API listing?

Reply:
done!